pull/context.go

```diff
@@ -76,6 +76,9 @@ type Context interface {
 	// Reviews lists all reviews on a Pull Request. The review order is
 	// implementation dependent.
 	Reviews() ([]*Review, error)
+
+	// IsDraft returns the draft status of the Pull Request.
+	IsDraft() *bool
```

Comment:
What's the reason for this being a `*bool` instead of `bool`? Is it safe to set it to false if we can't read it from GitHub for some reason?

Reply:
Good point, its not necessary to use a `*bool` here